docs/user-guide/configuration/externalConfig.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: main
+title: External Config
+category: User Guide
+menu: menu
+toc:
+    - title: External Config
+      url: "#external-config"
+    - title: Configure external config in parent pipeline
+      url: "#configure-external-config-in-parent-pipeline"
+    - title: Parent and Child Relationship
+      url: "#parent-and-child-relationship"
+    - title: Parent and Child Relationship
+      url: "#parent-and-child-relationship"
+
+---
+# External Config
+External config allows a single parent pipeline to create and manage build configuration of other "child" pipelines with a single `screwdriver.yaml`. The source code for each child pipeline will be taken from each child repository.
+
+This feature will allow users who own multiple  source code repositories with the same workflow to manage their pipelines more easily.
```

Comment:
This feature allows easier management of multiple repositories with the same workflow.